source/jormungandr/jormungandr/interfaces/v1/serializer/pt.py

```diff
@@ -399,14 +399,13 @@ class PlaceSerializer(PbGenericSerializer):
     stop_area = StopAreaSerializer(display_none=False)
     stop_point = StopPointSerializer(display_none=False)
     administrative_region = AdminSerializer(display_none=False)
-    embedded_type = EnumField(attr='embedded_type', pb_type=NavitiaType, display_none=True)
+    embedded_type = EnumField(attr='embedded_type', required=False, pb_type=NavitiaType, display_none=True)
     address = AddressSerializer(display_none=False)
     poi = PoiSerializer(display_none=False)
-
+    distance = jsonschema.IntField(required=False, display_none=True, description='Distance to the object in meters')
```

Comment:
In `PlaceNearby` it was a str (yes that's ugly...). Remember that we can't change the interface, so if it was an `str` before it must stay one.
In `/place_nearby` it's an str:
```
{
    "distance": "104",
    "embedded_type": "poi",
    "id": "poi:osm:node:3889022681"
    "name": "Parking vélo (Paris)",
    "poi": ⊕{8 items},
    "quality": 0,
}
```

In `/places` it's an int...
```
{
    "address": ⊕{6 items},
    "distance": 4651,
    "embedded_type": "address",
    "id": "2.37715;48.846781"
    "name": "20 Rue Hector Malot (Paris)",
    "quality": 0,
}
```